### PR TITLE
updated bcn microsite

### PR DESCRIPTION
**Pull request by Klarala13**

### CODE REVIEW

File: barcelona.html [hunk 1]
```diff
@@ -533,7 +536,7 @@ <h3>Follow us</h3>
 
       <div class="flex flex-col gap-5 text-lg lg:text-xl text-slate-100">
         <h3>Get in touch</h3>
-        <p><a href="mailto:hello@codebar.io">hello@codebar.io</a></p>
+        <p><a href="mailto:hello@codebar.io">barcelona@codebar.io</a></p>
```
Comment (KimberleyCook) [inline barcelona.html]: oh the hello@codebar.io needs to change here too.

File: css/output.css [hunk 2]
```diff
@@ -607,10 +599,22 @@ video {
   margin-bottom: 2.5rem;
 }
 
+.mb-12 {
```
Comment (KimberleyCook) [inline css/output.css:602]: @Klarala13 did you add these or was it automatic? We're using a css framework called Tailwind CSS that has these by default, you then just add them as css classes.

File: barcelona.html [hunk 3]
```diff
@@ -19,14 +19,14 @@
...
-  <meta name="twitter:title" content="uncodebar Barcelona 2023 | Saturday 11th November" />
+  <meta name="twitter:title" content="uncodebar Barcelona 2024 | Saturday 9th November" />
   <meta name="twitter:description"
     content="uncodebar is codebar's annual unconference. It's a day where our community comes together, shares knowledge, and strengthens the community that supports our workshops." />
   <meta name="twitter:image" content="./images/social-banner.png" />
 
   <!-- fb, linkedin, etc -->
   <meta property="og:type" content="article" />
-  <meta property="og:title" content="uncodebar Barcelona 2023 | Saturday 11th November" />
+  <meta property="og:title" content="uncodebar Barcelona 2023 | Saturday 9th November" />
```
Comment (KimberleyCook) [inline barcelona.html]: Oops there is a 2023 still here

File: barcelona.html [hunk 4]
```diff
@@ -253,18 +253,21 @@ <h2 class="lg:text-5xl text-2xl font-semibold">uncodebar highlights</h2>
...
         <div class="flex flex-wrap">
 
+          <div class="md:w-1/4"></div>
+
           <div class="md:w-2/4 flex w-screen flex-col lg:p-20 p-10 items-center">
-            <img class="pb-5 max-h-28" src="images/contentful.png" alt="a logo of contentful">
-            <p class="text-center text-lg max-w-md font-semibold mb-10">More than a headless CMS, Contentful is the
-              API-first composable content platform to create, manage and publish content on any digital channel.</p>
-            <a href="https://www.contentful.com/" class="bg-gradient-to-tr from-button_blue to-button_blue_ py-5 px-9 rounded-full text-white" target="_blank">FIND OUT MORE</a>
+            <img class="pb-5 max-h-28" src="images/travelperk-logo.jpeg" alt="a logo of TravelPerk">
```
Comment (KimberleyCook) [inline barcelona.html:259]: You've change this image URL but I don't see the travelperk image in the pull request

Comment (Klarala13) [reply barcelona.html:259]: It was already on the repo, so I just used that one and their description text from their website

File: css/output.css (hunk 2 above)
Comment (Klarala13) [reply css/output.css:602]: It was automatic, I use tailwind but when I did the install, they appeared. Can delete them if you think.

File: barcelona.html (hunk 1 above)
Comment (Klarala13) [reply barcelona.html]: fixed :)

File: barcelona.html (hunk 3 above)
Comment (Klarala13) [reply barcelona.html]: fixed :)

File: css/output.css (hunk 2 above)
Comment (KimberleyCook) [reply css/output.css:602]: ok cool, no no it's cool that was what I wanted to check. Thanks :)

File: barcelona.html (hunk 4 above)
Comment (KimberleyCook) [reply barcelona.html:259]: Ah yeah course, because of London 🤦‍♀️

**Comment by KimberleyCook** [comment]
@Klarala13 Niiice, thanks for updating the page. It's merged in now :)

**Pull request merged**